src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/specify-when-alerts-create-incidents.mdx

```diff
@@ -26,14 +26,14 @@ import alertsByCondition from 'images/alerts_diagram_by-condition.webp'
 import alertsBySignal from 'images/alerts_diagram_by-signal.webp'
 
 <Callout variant="important">
-  In this document, we use the new terms "issue" and "incident." Please see [this article](https://discuss.newrelic.com/t/new-terminology-coming-for-alerts/163727) for a full explanation of these new terms.
+  In this document, we use the new terms "issue" and "incident." Please see [this article](https://discuss.newrelic.com/t/new-terminology-coming-for-alerts/163727) for a full explanation of these terms.
 </Callout>
 
 An issue is a collection of one or more incidents.
 
-Issues determine when you are notified about incidents disrupting your business. These incidents occur when your given alert conditions (like CPU overage, synthetic monitoring crashes, or slow web app response times) are met.
+Issues determine when you're notified about incidents disrupting your business. These incidents occur when your given alert conditions (like CPU overage, synthetic monitoring crashes, or slow web app response times) are met.
```

Comment:
This sentence feels a little awkward with the parentheses in the middle--it makes it seem like "your" is a typo. What about explaining alert conditions and how they relate to incidents? I think @HenryTech has a really good analogy for this.